pkg/serviced-fstrim

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env bash
+
+# Copyright 2016 The Serviced Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");              
+# you may not use this file except in compliance with the License.             
+# You may obtain a copy of the License at                                      
+#                                                                              
+#     http://www.apache.org/licenses/LICENSE-2.0                               
+#                                                                              
+# Unless required by applicable law or agreed to in writing, software          
+# distributed under the License is distributed on an "AS IS" BASIS,            
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.        
+# See the License for the specific language governing permissions and          
+# limitations under the License. 
+
+# this convenient script is intended to be run as SU by a cron job.
+# It discards the blockes unused by the mounted dfs volumes of serviced
+# It should not run more often than necessary.
+# Please refer to the description of fstrim man page.
+which fstrim>/dev/null 2>&1 || { echo "Error:fstrim not found! Please install." >&2; exit 1; }
+
+# capture block devices of the related serviced volumes
+blk_devs=$(lsblk -o MOUNTPOINT,TYPE | grep '/opt/serviced/' | grep ' dm' | sort | uniq)
```

Comment:
I suggest you use `lsblk --output` to specify the columns you want to parse, then get them directly. Merely searching for the string " dm" is too easy to fail.